sql/ddl.md

```diff
@@ -226,6 +226,52 @@ RENAME TABLE
 ALTER TABLE old_table RENAME new_table;
 ```
 
+## CREATE INDEX 语法
+
+```sql
+CREATE [UNIQUE] INDEX index_name
+    [index_type]
+    ON tbl_name (index_col_name,...)
+    [index_option] ...
+
+index_col_name:
+    col_name [(length)] [ASC | DESC]
+
+index_option:
+    KEY_BLOCK_SIZE [=] value
+  | index_type
+  | COMMENT 'string'
+
+index_type:
+    USING {BTREE | HASH}
+```
+
+[CREATE INDEX](#create-index-语法) 语句的作用是为一个已经存在的表创建一个索引。在功能上，[CREATE INDEX](#create-index-语法) 对应于 [ALTER TABLE](#alter-table-语法) 语句的创建索引功能。与 MySQL 一样 [CREATE INDEX](#create-index-语法) 不能创建主键索引。
```

Comment:
Bare Backquote for code formatting is enough.
Is full level of cross reference required? 
If so, markdown shouldn't be used

Reply:
It is the same style as MySQL create index document.